reliability-v2/tasks/TaskManager.py

```diff
@@ -134,11 +133,16 @@ def run_users_tasks(self,group):
         users_tasks = []
         # todo: validate
         if persona == "admin":
+            users = group.get("uses",1)
```

Comment:
yes, thanks

Reply:
I updated. Actually this line is not used. Because currently for admin we have only one kubeadmin user, to avoid conflict admin operation with multiple admin login session, only 1 user is supported now. I added some comments to clarify.